imports/plugins/core/core/server/startup/accounts.js

```diff
@@ -1,12 +1,12 @@
 import _ from "lodash";
-import Hooks from "@reactioncommerce/hooks";
 import Logger from "@reactioncommerce/logger";
 import Random from "@reactioncommerce/random";
 import { Meteor } from "meteor/meteor";
-import ReactionError from "@reactioncommerce/reaction-error";
 import { Accounts } from "meteor/accounts-base";
 import * as Collections from "/lib/collections";
```

Comment:
Should we update this to use the `rawCollections`, along with lines 160 & 162?

I also have this question on any file that imports `lib/collections`, which is a lot of them in this PR, just didn't want to overkill and comment on each on.

Reply:
This file is Meteor code (assumes it's running in Fibers), so I think it's best to keep using the Meteor collections from `/lib/collections` in such code until we convert it all. You absolutely could import and use `rawCollections` instead, but then you'd need to wrap every call in `Promise.await` because all of that non-Meteor stuff uses Promises instead of Fibers. (`Promise.await` is Meteor's interface between Promises and Fibers. They're just two different ways of making async JavaScript appear to be synchronous.)